everviz/data/load_csv/get_data.py

```diff
@@ -0,0 +1,9 @@
+import pandas as pd
+from webviz_config.common_cache import CACHE
+from webviz_config.webviz_store import webvizstore
+
+
+@CACHE.memoize(timeout=CACHE.TIMEOUT)
+@webvizstore
+def get_data(data_path) -> pd.DataFrame:
```

Comment:
I almost feel a bit sorry for this alone annotation, there are no one else :stuck_out_tongue: